runtime/doc/dev_vim_patches.txt

```diff
@@ -0,0 +1,303 @@
+*dev-vim-patches.txt*                            Merging patches from upstream
+
+                            NVIM REFERENCE MANUAL
+
+See |dev-vim-patches-quickstart| to get started immediately. Also note the
+section on Not Applicable patches.
+
+Nvim was forked from Vim 7.4.160; it is kept up-to-date with relevant Vim
+patches in order to avoid duplicate work. Run `vim-patch.sh`
+https://github.com/neovim/neovim/blob/master/scripts/vim-patch.sh to see the
+status of Vim patches:
+
+>bash
+    ./scripts/vim-patch.sh -l
+<
+
+Everyone is welcome to |dev-vim-patches-pull-requests| for relevant Vim
+patches, but some types of patches are |dev-vim-patches-not-applicable|.
+
+==============================================================================
+QUICKSTART                                        *dev-vim-patches-quickstart*
+
+1. Pull down the Nvim source:
+>bash
+    git clone https://github.com/neovim/neovim.git
+<
+2. Run `./scripts/vim-patch.sh -l` to see the list of missing Vim patches.
+
+3. Choose a patch from the list (usually the _oldest_ one), e.g. `8.0.0123`.
+
+- Check for open vim-patch PRs
+  https://github.com/neovim/neovim/pulls?q=is%3Apr+is%3Aopen+label%3Avim-patch.
+
+4. Run `./scripts/vim-patch.sh -p 8.0.0123`
+
+5. Follow the instructions given by the script.
+
+NOTES ~
+
+- It's strongly recommended to work on the _oldest_ missing patch, because
```

Comment:
```suggestion
- It's strongly recommended to work on the oldest missing patch, because
```
and similar (no markup in vimdoc, especially not with `*`). I can go over the file if you like.

Reply:
fixed them all